helpers.sh

```diff
@@ -54,7 +54,13 @@ function repo_sync {
 		status "Adding git config extraheader for $domain"
 		git config --global http.https://${domain}.extraheader "$(cat /secrets/git.http.extraheader)"
 	fi
-	run repo init --repo-rev=v2.29.4 --no-clone-bundle -u $* ${REPO_INIT_OVERRIDES}
+	for i in $(seq 4); do
+		run repo init --repo-rev=v2.29.4 --no-clone-bundle -u $* ${REPO_INIT_OVERRIDES} && break
+		run sleep $(($RANDOM%20+10))
+		if [ $i -eq 4 ] ; then
+			exit 1
```

Comment:
I don't think we need to use the "run" helper when calling sleep. We also don't need to sleep when $i=4. So maybe more like:
```
                run repo init --repo-rev=v2.29.4 --no-clone-bundle -u $* ${REPO_INIT_OVERRIDES} && break
		[ $i -eq 4 ] && exit 1
                status "repo init failed, sleeping and trying again"
                sleep $(($RANDOM%20+10))
```
Also - does this mean we could sleep up to 30s?  Maybe sleep more like `i*2` and you could add some small millesecond jitter if you are really excied.